lib/SIL/SILProfiler.cpp

```diff
@@ -101,6 +101,22 @@ static void walkForProfiling(ASTNode N, ASTWalker &Walker) {
 }
 
 SILProfiler *SILProfiler::create(SILModule &M, ASTNode N) {
+  // Unless there are no primary inputs (WMO), code outside of a primary input
+  // may not be fully formed. Defer instrumenting non-primary files, with the
+  // expectation that that code will be visited later.
+  const auto &Opts = M.getOptions();
```

Comment:
I don't understand why Opts is moving here.

Reply:
This was accidental code movement, sorry about that.